src/libraries/Microsoft.Extensions.DependencyModel/tests/Microsoft.Extensions.DependencyModel.Tests.csproj

```diff
@@ -13,6 +12,7 @@
     <PackageReference Include="FluentAssertions" Version="4.19.4" />
     <PackageReference Include="Moq" Version="$(MoqVersion)" />
     <PackageReference Include="Microsoft.DotNet.ProjectModel" Version="1.0.0-rc2-002702" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
```

Comment:
This shouldn't be necessary. The unit test doesn't need Newtonsoft.Json.

Reply:
From which?

Reply:
It was failing without it.

Reply:
The ProjectModel directly above it references Newtonsoft.

Reply:
Sorry I should have read the error message more carefully. The build didn't fail because of the missing dependency but because of version 7.0.1 of `Newtonsoft.Json` is being resolved via `PackageTargetFallback` as that version isn't compatible with .NETCoreApp. Hence I'm upgrading the version to avoid the `PackageTargetFallback`.

Reply:
```
F:\workspace\_work\1\s\src\libraries\Microsoft.Extensions.DependencyModel\tests\Microsoft.Extensions.DependencyModel.Tests.csproj : error NU1701: Package 'Newtonsoft.Json 7.0.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETCoreApp,Version=v5.0'. This package may not be fully compatible with your project. [F:\workspace\_work\1\s\Build.proj]
```

Reply:
This hasn't happened before because we were not restoring for .NET Framework: https://github.com/dotnet/runtime/blob/master/src/libraries/tests.proj#L6.

Reply:
Ok, makes sense. I hope to rewrite some of these tests and remove the ProjectModel dependency anyway, so this will get removed when that happens.